local-cluster/src/local_cluster.rs

```diff
@@ -696,7 +699,8 @@ impl Cluster for LocalCluster {
     fn get_validator_client(&self, pubkey: &Pubkey) -> Option<ThinClient> {
         self.validators
             .get(pubkey)
-            .map(|f| create_client(f.info.contact_info.client_facing_addr()))
+            .map(|f| f.info.contact_info.client_facing_addr())
+            .map(|(rpc, tpu)| create_client(rpc, tpu))
```

Comment:
```suggestion
            .map(|f| {
                let (rpc, tpu) = f.info.contact_info.client_facing_addr();
                create_client(rpc, tpu)
            })
```

Reply:
Thank you @t-nelson ! By the way, is the coverage test case `bank::tests::test_remove_unrooted_scan_then_recreate_same_slot_before_scan`'  related to the issue? I get `panicked at 'test timed out', runtime/src/bank.rs:14633:9`. Not sure why I get this error :(

Reply:
should be unrelated.  I've restarted the job

Reply:
Thank you @t-nelson! I get a different time out error

test validator::tests::validator_exit has been running for over 60 seconds
Then # Received cancellation signal, interrupting

I noticed the tests are kill around one hour time frame. It seems like there is a time limit to run all the tests and the job gets killed if over the time limit?

Reply:
yes, this test is known flaky. we'll retry our way to victory in the meantime